docs/EVG.md

```diff
@@ -490,6 +491,8 @@ Country:
 
 **Contents:**    These fields MUST NOT contain information that is not relevant to the level of the Incorporating Agency or Registration Agency.  For example, the Jurisdiction of Incorporation for an Incorporating Agency or Jurisdiction of Registration for a Registration Agency that operates at the country level MUST include the country information but MUST NOT include the state or province or locality information.  Similarly, the jurisdiction for the applicable Incorporating Agency or Registration Agency at the state or province level MUST include both country and state or province information, but MUST NOT include locality information.  And, the jurisdiction for the applicable Incorporating Agency or Registration Agency at the locality level MUST include the country and state or province information, where the state or province regulates the registration of the entities at the locality level, as well as the locality information.  Country information MUST be specified using the applicable ISO country code.  State or province or locality information (where applicable) for the Subject's Jurisdiction of Incorporation or Registration MUST be specified using the full name of the applicable jurisdiction.
 
+Effective as of 1 October 2020, the CA SHALL ensure that the information to be included is consistent with its publicly-available disclosure, as described in Section 11.1.3.
```

Comment:
I'm not sure I like "information to be include is *consistent with*".  It seems like a source of many future disagreement.  I think I'd prefer something that expresses more explicitly the fact that information sources used must have been disclosed.

Reply:
Sure, but this is about the jurisdiction fields.

If I disclose Agency X as being relevant to Jurisdiction Y, but then issue a cert using information from Agency X and state Jurisdiction Z, then the fact the X is disclosed isn’t sufficient, because it was only disclosed for Y, and not Z.

This is trying to capture that the information in the cert must match how the CA said, via their disclosure, how they put the information in.

Reply:
Yes, if that's what you want, I think it needs to be explicitly said.  Let me think about some language...

Reply:
@timfromdigicert I guess I'm not sure I understand an example of what you think "consistent with" may cause trouble. However, in the spirit of more words to say less, what do you think?

```suggestion
Effective as of 1 October 2020, prior to a certificate being issued, the CA SHALL ensure the information to be included within these fields is identical to the information disclosed within the current version of the CA's publicly-available disclosure of values for these fields, as described in Section 11.1.3.
```

Reply:
There are a couple problems with that, including the fact that the information would only have to be consistent with a disclosed source, which is not necessarily the appropriate source or the source being used.

I think we can explicitly list the requirements (there aren't many):

"..., prior to a certificate being issued, the CA SHALL ensure its latest publicly available disclosure of sources (see Section 11.1.3) contains a source such that:

(1) the source lists the entity as being registered in that jurisdiction,
(2) the certificate fields `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) have one of the acceptable values listed for that source in the disclosure, and
(3) the Registration Number matches one of the accepted or allowed syntaxes (if disclosed)."

Or something similar.  Just to be explicitly clear of exactly what is required.

Reply:
@timfromdigicert I'm not sure how to reach the interpretation you're offering? The CA would have to ignore a variety of other sections (e.g. 9.2.5, 11.2.1) to reach that conclusion, right? The EVGs are clear that there is only one applicable Agency.

Your (1) is already stated elsewhere (e.g. 11.2.1 (1)), so I'm loathe to repeat it.
Your (3) seems like it would belong in 9.2.5

Let me try a different approach, to see if we're getting closer:
```suggestion
Effective as of 1 October 2020, the CA SHALL ensure that, at time of issuance, the values within these fields have been disclosed within the latest publicly-available disclosure, as described in Section 11.1.3, as acceptable values for the applicable Incorporating Agency or Registration Agency.
```

"these fields" matches the opening sentence of "contents". "applicable Incorporating Agency or Registration Agency" matches the third and fourth sentences.

And then something similar for 9.2.5?

Reply:
That approach is fine with me, too.